packages/ontario-design-system-complete-styles/package.json

```diff
@@ -16,27 +16,26 @@
 		"clean": "rm -rf dist",
 		"clean:full": "rm -rf node_modules && npm run clean",
 		"format": "prettier --write .",
-		"scss": "node-sass --importer node_modules/node-sass-package-importer/dist/cli.js --watch scss -o css",
+		"scss": "sass --watch scss:css",
 		"docs:copy:readme": "ts-node ../../scripts/documentation-helper.ts -f README.md -p \"---\ntitle: Complete Styles\n---\n\" -df ../app-web-components-documentation/docs/06-readme-complete-styles.md"
 	},
 	"dependencies": {
-		"@ongov/ontario-design-system-component-library": "5.0.0-alpha.1",
-		"@ongov/ontario-design-system-design-tokens": "4.3.1-alpha.1",
-		"@ongov/ontario-design-system-global-styles": "4.3.1-alpha.1",
-		"gulp-flatten": "^0.4.0"
+		"@ongov/ontario-design-system-component-library": "4.3.0",
```

Comment:
We'll want to merge in `develop` before we merge this into `develop` as these version numbers are now out of date.

_Note_: the `alpha` designations in the diff.